mkosi/distributions/fedora.py

```diff
@@ -57,13 +50,9 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str]) -> None:
             # In other versions, the "fedora" repo is frozen at release, and "updates" provides any new packages.
             updates_url = None
 
-        if releasever in FEDORA_KEYS_MAP:
-            gpgid = f"keys/{FEDORA_KEYS_MAP[releasever]}.txt"
-        else:
-            gpgid = "fedora.gpg"
-
         gpgpath = Path(f"/etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-{releasever}-{state.config.architecture}")
-        gpgurl = urllib.parse.urljoin("https://getfedora.org/static/", gpgid)
+        # https://fedoraproject.org/security/
+        gpgurl = urllib.parse.urljoin("https://getfedora.org/fedora.gpg")
```

Comment:
should this maybe just be
```suggestion
        gpgurl = "https://getfedora.org/fedora.gpg"
```

Reply:
Yeah, I was just about to push it :)